### PR TITLE
scripts: generate dynamic deploy pipeline for driver/linker/chromium

### PR DESCRIPTION
**Pull request by jazzdan**
Fixes TT-27

With the corresponding Buildkite pipeline definition:

```yaml
steps:
  - label: ":pipeline: Pipeline upload"
    command: "curl -s https://raw.githubusercontent.com/replayio/chromium/$BUILDKITE_BRANCH/replay_build_scripts/build-pipeline.py | python3 | buildkite-agent pipeline upload" 
    plugins:
      - thedyrt/skip-checkout#v0.1.1: ~
```

### CODE REVIEW

File: replay_build_scripts/build-pipeline.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+def generate_buildkite_pipeline(commit_hash):
+    """Generates a Buildkite pipeline with a dynamic commit hash in JSON format."""
+
+    # driver_revision is the first 12 characters of the commit hash.
+    driver_revision = commit_hash.split()[0][:12]
+    pipeline = {
+        "steps": [
+            {
+                "trigger": "deploy-driver-linker",
```
Comment (toshok) [inline replay_build_scripts/build-pipeline.py:18]: q here: do we even need a `deploy-driver-linker` pipeline anymore?  iirc that just makes it the default driver that's downloaded in `node build`?  Can we get by with triggering `build-driver-linker` instead?

**Review by toshok — APPROVED** [review]
lgtm, but a (offline) question re what it means to deploy driver/linker in the future

File: replay_build_scripts/build-pipeline.py (hunk 1 above)
Comment (jazzdan) [reply replay_build_scripts/build-pipeline.py:18]: Good point! Yes you're correct. I'll make that change.

File: replay_build_scripts/build-pipeline.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                "key": "deploy-driver-linker",
+                "build": {
+                    "commit": commit_hash,
+                    "message": "Triggered from chromium: ${BUILDKIT_MESSAGE}",
```
[the remark this answers is not in the document]
Comment (jazzdan) [reply replay_build_scripts/build-pipeline.py:22]: Man the buildkit vs buildkite thing is really the bane of my existence

**Pull request merged**